tests/import_export_tests.py

```diff
@@ -547,6 +550,34 @@ def test_import_druid_override_identical(self):
         self.assert_datasource_equals(
             copy_datasource, self.get_datasource(imported_id))
 
+    def test_export_dashboards_util(self):
```

Comment:
@mistercrunch @arpit-agarwal I've noticed this test failing the py36-postgres tests when a pr has not made any changes to this code. For example https://travis-ci.org/apache/incubator-superset/jobs/436740957 Can you take a look?

Reply:
@mistercrunch @arpit-agarwal this is blocking merging all PRs into master

Reply:
@mistercrunch @arpit-agarwal this flaky test is still failing most builds for `py36-postgres` therefore blocking merges. Can you either PTAL or we will revert this PR if it's not fixed by noon today PST?

cc @john-bodley @kristw @michellethomas @graceguo-supercat

Reply:
i made a test branch which reverted this whole PR, and it passed all CI tests. i will file PR to revert this. Thank you!

Reply:
@arpit-agarwal Thank you very much for investigation. After I reverted this PR, what we observed is, most PRs passed all CI tests, includes postgres one. But master branch still failed at postgres test. Right now i am totally confused what exactly happened :(

Reply:
As suggested we can look at the content of PR #5693  to fund the route cause.

I do see some changes in test in that PR related to test.
Any recent configruration changes on tarvis or postgress version or lsome python lib?